gatsby-config.js

```diff
@@ -399,6 +399,128 @@ module.exports = {
                       },
                     ],
                   },
+                  {
```

Comment:
Can I get a quick one liner that explains what these files/fields are defining ?